### PR TITLE
Add postToTumblr, postToWordpress & more actions

### PR DESCRIPTION
**Pull request by JB1905**
**Checks**

- [x] I've checked there are no linting errors.
- [x] I've checked the code is still at 100% test coverage.

**Added Actions (if relevant)**

- _postToTumblr_
- _postToWordpress_
- _postOnFacebook_
- _postOnInstagram_

### CODE REVIEW

File: src/actions/postOnInstagram.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+/**
+ * Opens the photo passed into this action in the Instagram app and copies the caption to the clipboard.
+ *
+ * ```js
+ * postOnInstagram();
+ * ```
+ */
+const postOnInstagram = (): WFWorkflowAction => ({
+  WFWorkflowActionIdentifier: 'com.burbn.instagram.openin',
+  WFWorkflowActionParameters: {},
```
Comment (joshfarrant) [inline src/actions/postOnInstagram.ts]: Could you add the `caption` param into this please, and add a test for it too?

File: src/actions/postOnFacebook.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { withActionOutput } from '../utils';
+
+import WFWorkflowAction from '../interfaces/WF/WFWorkflowAction';
+
+/**
+ * Shares the input on Facebook.
```
Comment (xAlien95) [inline src/actions/postOnFacebook.ts]: Could you edit this line to the following, please?
```
 * Post on Facebook Action. Shares the input on Facebook.
```

This should be added also to all the other actions you submitted.
In the end, each action should have a comment in this format:
```
 * {{Action name as shown in app}} Action. {{Description as shown in app}}.
```
This format will help getting the shown name in the next documentation website. Thanks!

Comment (JB1905) [reply src/actions/postOnFacebook.ts]: Added

File: src/actions/postOnInstagram.ts (hunk 1 above)
Comment (JB1905) [reply src/actions/postOnInstagram.ts]: Added

**Comment by joshfarrant** [comment]
Looks good, thanks!

**Pull request merged**